src/app/containers/Ad/Canonical/AdSlot.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { string } from 'prop-types';
+
+const AdSlot = ({ type }) => {
+  const location = useLocation();
```

Comment:
We don't expect this to work at the moment on client rendered routes. We can remove this.